scripts/test/system.sh

```diff
@@ -179,15 +185,14 @@ _subctl cloud prepare generic --kubecontext cluster1 && exit 1
 # Rename old broker-info.subm to be able to compare it with the recovered broker-info.sum
 echo "Renaming broker-info.subm to broker-info.subm.orig"
 mv "${DAPPER_SOURCE}"/output/broker-info.subm "${DAPPER_SOURCE}"/output/broker-info.subm.orig
+base64 -d "$DAPPER_SOURCE"/output/broker-info.subm.orig > "$DAPPER_SOURCE"/output/decoded_broker_info.subm.orig
 
 # Test subctl recover-broker-info invocations
 _subctl recover-broker-info --context cluster1
-cmp "${DAPPER_SOURCE}"/output/broker-info.subm.orig broker-info.subm
-rm -f broker-info.subm
+cmp_rm_files
```

Comment:
I'd combine it all to a single command `test_subctl_recover_broker_info` or some such name

Then moving old `broker-info.subm` and decoding it can be hidden inside as well, as it only pertains to this test case, making tests clearer